expression/builtin_control.go

```diff
@@ -15,12 +15,12 @@
 package expression
 
 import (
-	"github.com/cznic/mathutil"
 	"github.com/pingcap/tidb/parser/mysql"
 	"github.com/pingcap/tidb/sessionctx"
 	"github.com/pingcap/tidb/types"
 	"github.com/pingcap/tidb/types/json"
 	"github.com/pingcap/tidb/util/chunk"
+	mathutil "github.com/pingcap/tidb/util/math"
```

Comment:
Maybe we should be consistent and use either a unique package name, so we don't need an alias, or use a unique alias so it is the same everywhere?
So far I have seen 'math', 'tidbmath' and 'mathutil'. Where 'math' is too easy to be taken as the standard 'math' package.
I propose to have the name mathutil, but are open to other names as well, just as long as it is not colliding with others and can be used consistently.

Reply:
I have renamed ```util/math``` to ```util/mathutil```. and remove ```mathutil``` third library.